lib/rdoc/options.rb

```diff
@@ -1116,6 +1122,15 @@ def parse argv
 
       opt.separator nil
 
+      opt.on(
+        "--server[=PORT]",
+        Integer,
+        "[Experimental] Run WEBrick server with generated documentation.",
```

Comment:
I feel we can remove the `[Experiment]` flag, we're not shipping something we're not expecting will work or introduce any breaking changes.

Reply:
My interpretation of "experimental" is: the feature works but may not live up to expectation at the moment and has a slight chance to be removed if it's proven problematic.

In this case, I don't expect it to be removed later but it doesn't have the maturity to work like `yard server` or some static site generators' server mode.